.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: go
+go:
+- 1.7.4
+before_install:
+- go get github.com/zmap/zcertificate
```

Comment:
Can these be summed up to `go get ./...`?

Reply:
That would definitely be the ideal: keeping the dependencies in sync in two different locations is going to be mistake-prone...but, I was having some issues with `go get ./...` -- though to be fair I gave up pretty quickly on trying to fix it properly when I saw that zdns gave an explicit list, too.

If I can get travis running locally, I'll see if I can get it working.

Reply:
https://github.com/zmap/zlint/blob/master/.travis.yml might be a better reference.

Reply:
I think the problem was some bad paths -- https://github.com/zmap/zgrab2/pull/23/commits/577e79cb54c9d9b25be285a0047919423c643650 seems to have fixed it so that go get ./... works.